src/components/TableDialog.vue

```diff
@@ -72,6 +72,20 @@
               />
             </v-col>
           </v-row>
+
+          <v-row>
+            <v-col
+              v-for="(type, field) in columnType"
+              :key="field"
+              cols="2"
+            >
+              <v-select
+                v-model="columnType[field]"
```

Comment:
```suggestion
                v-model="columnType[field]"
                outlined
```

Reply:
Add outlined prop like all the other text fields in the app

Reply:
Thanks for this; updated.